README.md

```diff
@@ -133,7 +133,7 @@ While Transcribe provides real-time transcription and optional response suggesti
 Incorrect API key provided: API_KEY. You can find your API key at https://platform.openai.com/account/api-keys.
 ```
 
-**Models**: The default install of transcribe has the tiny(72 Mb) model. base (138 Mb), small (461 Mb) models can be downloaded and used for transcription by following instructions using transcribe command line. The larger models provide better quality transcription and they have higher memory requirements.
+**Models**: The default install of transcribe has the tiny(72 Mb) model for english. Other larger models and their multi lingual versions can be downloaded and used for transcription by following instructions using transcribe command line arguments. Larger models provide better quality transcription and they have higher memory requirements.
```

Comment:
In the section of enhancements from base repo, can add this functionality as well.
It is a great add and was requested some time back.

Reply:
Good point. Updated the readme.